cmd/typesctl/providers/azure_types.go

```diff
@@ -151,13 +50,5 @@ func generateTypesAzure() error {
 		return fmt.Errorf("unable to generate types: %w", err)
 	}
 
```

Comment:
Why we don't want it as part of the interface as well?

Reply:
The `typesctl` is a quick and dirty code, it uses three function pointers, feel free to refactor this if you want.